test/src/sdk-harness/test_artifacts/storage_vec/svec_u8/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "svec_u8"
+version = "0.1.0"
```

Comment:
Unless there is something to add to the conversation you do not need to respond to each comment. You can click "resolve conversation" and that's enough otherwise every comment results in an email.